src/Page.js

```diff
@@ -594,6 +594,12 @@ Page.prototype.insertHeaderFile = function (pageData) {
 Page.prototype.insertFooterFile = function (pageData) {
   const { footer } = this.frontMatter;
   let footerFile;
+
+  if (footer === 'none') {
```

Comment:
if 'none' is a keyword, put it as a constant.

Reply:
will make the change 👍